jbot/src/main/java/me/ramswaroop/jbot/core/slack/Bot.java

```diff
@@ -169,6 +169,8 @@ public final void handleTextMessage(WebSocketSession session, TextMessage textMe
             if (event.getType() != null) {
                 if (event.getType().equalsIgnoreCase(EventType.IM_OPEN.name())) {
                     slackService.addDmChannel(event.getChannelId());
+                } else if (event.getType().equalsIgnoreCase(EventType.IM_CREATED.name())) {
+                    slackService.addDmChannel(event.getChannel().getId());
```

Comment:
Yes, we can do that. With channelId in the outer object having higher preference than the inner one.